lib/public/views/QcFlags/details/qcFlagDetailsComponent.js

```diff
@@ -50,27 +50,23 @@ const verifyButton = (qcFlagDetailsModel) => {
         qcFlag: remoteQcFlag,
         verificationEnabled,
     } = qcFlagDetailsModel;
+
+    const verificationButton = h(
+        'button#verify-qc-flag.btn.btn-primary',
+        !verificationEnabled
+            ? {
+                onclick: () => {
+                    qcFlagDetailsModel.verificationEnabled = true;
+                },
+            }
+            : { disabled: true },
+        'Verify',
+    );
```

Comment:
You create a component even when it's not necessary. Either put it directly inline, or make it a function

Reply:
put inline